README.md

```diff
@@ -26,26 +26,13 @@ Adds a set of popular Google Web Fonts to Design Palette Pro. Provides the prope
 
 ## Frequently Asked Questions ##
 
-### What is this? ###
-
-The description covers it all.
-
 ### Why Doesn't This Work On My Theme? ###
 
 Because this is an add-on for [Design Palette Pro](http://genesisdesignpro.com/ "Genesis Design Palette Pro"). Without that plugin, nothing will work.

```

Comment:
Revise this to:

This plugin only works with [Design Palette Pro](https://genesisdesignpro.com/ "Design Palette Pro") and a [supported Genesis child theme](https://genesisdesignpro.com/themes/)